docs/reference/add_response_headers.md

```diff
@@ -18,6 +20,11 @@ add_response_headers:
   x-test-proto: "%PROTOCOL%"
   x-test-ip: "%DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT%"
   x-test-static: This is a test header
+  x-test-append:
+    value: This header will replace if any other present
+    append: False # Values will be overriden
```

Comment:
I think this will be better explained in the docs above, rather than explaining in a comment. Docs are a bit unclear, would be great if you could add a line or two about this feature ;)

Reply:
Already added, this was supposed to be an inline notion about it. But I feel it is repetitive and removed the comments.